addons/account/models/account.py

```diff
@@ -1375,29 +1375,40 @@ def _check_repartition_lines(self, lines):
         base_line = lines.filtered(lambda x: x.repartition_type == 'base')
         if len(base_line) != 1:
             raise ValidationError(_("Invoice and credit note repartition should each contain exactly one line for the base."))
+        tax_lines = lines - base_line
+        if not tax_lines:
+            if base_line.factor_percent == 100:
+                values = {
+                    'repartition_type': 'tax',
+                    'factor_percent': 100.0,
+                    'company_id': base_line.company_id.id,
+                    'country_id': base_line.country_id.id,
+                }
+                self.env['account.tax.repartition.line'].create([
+                    {**values, 'invoice_tax_id': self.id},
+                    {**values, 'refund_tax_id': self.id},
```

Comment:
This is not great given the API but I would also need to change the API to make it better...